src/megaapi_impl.cpp

```diff
@@ -23267,7 +23267,7 @@ char *MegaTCPServer::getLink(MegaNode *node, string protocol)
     allowedHandles.insert(lastHandle);
 
     ostringstream oss;
-    oss << protocol << (useTLS ? "s" : "") << "://127.0.0.1:" << port << "/";
+    oss << protocol << (useTLS ? "s" : "") << "://[::1]:" << port << "/";
```

Comment:
This causes issues in non IPv6 supportive clients: e.g: gvsfd-dav in Ubuntu 18.04. 
Browsers & other common clients (e.g vlc) seem to be good with it for local interface address (although fail to use scoped local links).  Will need some testing in different Windows versions in MEGAsync & probably some wording in MEGAcmd inner docs to alert possible issues

Reply:
We could add a parameter to the `getLink` function to return either an IPv4 URL or an IPv6 one, default IPv4 to keep compatibility with those clients.